src/fXmlRpc/Parser/NativeParser.php

```diff
@@ -31,16 +30,28 @@
 
 final class NativeParser implements ParserInterface
 {
-    public function __construct()
+    /**
+     * @var bool
+     */
+    private $validateResponse;
+
+    public function __construct($validateResponse = true)
     {
         if (!extension_loaded('xmlrpc')) {
             throw MissingExtensionException::extensionMissing('xmlrpc');
         }
+        $this->validateResponse = $validateResponse;
     }
 
     /** {@inheritdoc} */
     public function parse($xmlString, &$isFault)
     {
+        libxml_use_internal_errors(true);
```

Comment:
Given the XmlChecker should be self contained, I think we need to duplicate it. Also, we need to properly reset it afterwards, otherwise we change global state. Example:

```php
$useErrors = libxml_use_internal_errors(true);
… Do something …
libxml_use_internal_errors($useErrors);
```